AGAMI/Sources/Service/FirebaseAuthService.swift

```diff
@@ -89,4 +105,77 @@ final class FirebaseAuthService {
             completion(.failure(signOutError))
         }
     }
+
+    func deleteAccount() async -> Bool {
+        guard let user = user else { return false }
+        guard let lastSignInDate = user.metadata.lastSignInDate else { return false }
+        let needsReauth = !lastSignInDate.isWithinPast(minutes: 5)
```

Comment:
```suggestion
        let needsReAuth = !lastSignInDate.isWithinPast(minutes: 5)
```
큰 건 아니지만, 카멜 케이스 지키는 쪽이 가독성이 더 좋을 것 같아용

Reply:
수정하겠습니답